### DISCUSSION TITLE
İBXC Stake

### DISCUSSION THREAD

**Discussion by Bulent3407** [IBAX-io/go-ibax#1623] [Soru Cevap - Türkçe] [answered]
İbax network üzerinde stake işlemleri olacakmı?Ben açıkçası airdroptan veya ön satıştan alacak olduğum ibxc coinlerimi uzun bir süre satmak istemiyorum hold ettiğimiz sürede stake ödüllerinin veya komisyonlarımızın olması çok iyi olurdu

**Comment by Abdullahoda** [comment, marked as answer]
Madencilik ve stake entegre edilecek

**Reply by THECROVV** [reply]
Süper

**Comment by Bulent3407** [comment]
#

**Reply by THECROVV** [reply]
Bende aynı düşüncedeyim, tutmak istiyorum iyi olmuş

**Reply by Bulent3407** [reply]
👍👍

**Comment by Eddyhunter07** [comment]
Olacak

**Reply by Bulent3407** [reply]
Olması iyi oldu
